### ISSUE TITLE
Exercise and associated dietary extremes impact on gut microbial diversity.

### ISSUE THREAD

**Issue by pschloss** [SchlossLab/march_madness_2015#7]
Exercise and associated dietary extremes impact on gut microbial diversity.
Clarke SF, Murphy EF, O'Sullivan O, Lucey AJ, Humphreys M, Hogan A, Hayes P, O'Reilly M, Jeffery IB, Wood-Martin R, Kerins DM, Quigley E, Ross RP, O'Toole PW, Molloy MG, Falvey E, Shanahan F, Cotter PD.
Gut. 2014 Dec;63(12):1913-20. doi: 10.1136/gutjnl-2013-306541. Epub 2014 Jun 9.
http://www.ncbi.nlm.nih.gov/pubmed/25021423

**Comment by nbaxter13** [comment]
Clarke and colleagues were interested in the effects of exercise on the gut microbiome. They hypothesized (for no apparent reason) that athletes would have a more diverse microbiome. They collected stool, blood, food questionnaires, and body composition scans from 40 elite Irish rugby players and 46 controls.  The overall structure of the microbiome was different between athletes and controls, and athletes had significantly higher alpha-diversity than controls.  Athletes also had higher caloric intake (especially in the form of whey protein supplements) and higher plasma creatine kinase levels.  Not surprisingly then, there was a very weak but significant correlation between diversity and creatine kinase levels, leading the authors to conclude that “diet and exercise are drivers of bacterial diversity in the gut”, and that “exercise increases gut microbial diversity in humans”.  They also observed an increase in Akkermansia in lean controls compared to obese controls, supporting previous findings in which Akkermansia is negatively correlated with obesity.  

All in all, the experimental design and methods of this study are somewhat lacking.  Their claim that exercise increases diversity would be fascinating and innovative, but there are far to many confounders in their data to back up that claim.  It seems like they may just as likely be detecting the effect of protein supplements or a group of men in a similar environment, with similar diets, being in VERY close physical contact.

tl;dr -  Being a rugby player at spring training camp and eating lots of whey protein makes your microbiome more diverse.

Overall – 4
Importance – 3
Innovation – 6
